staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go

```diff
@@ -37,6 +37,7 @@ func Resource(resource string) schema.GroupResource {
 	return SchemeGroupVersion.WithResource(resource).GroupResource()
 }
 
+// SchemeBuilder - initializes a scheme builder
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
```

Comment:
should the comment be above `SchemeBuilder` here?

Reply:
Moved the comment to the line just before.